README.md

```diff
@@ -32,6 +32,7 @@ Below is a listing of access used.
 * [Foreman infrastructure access](https://theforeman.github.io/foreman-infra/#access) including [secrets](https://theforeman.github.io/foreman-infra/secrets/)
 * [Jenkins access](https://theforeman.github.io/foreman-infra/jenkins/#access)
 * [Koji access](https://theforeman.github.io/foreman-infra/koji/#using-koji-as-a-user)
+* [web01 stagingyum acess](https://github.com/theforeman/foreman-infra/blob/master/puppet/modules/web/manifests/vhost/stagingyum.pp#L9)
```

Comment:
```suggestion
* [web01 stagingyum access](https://github.com/theforeman/foreman-infra/blob/master/puppet/modules/web/manifests/vhost/stagingyum.pp#L9)
```

And added as a member of https://copr.fedorainfracloud.org/groups/g/theforeman/coprs/?

Reply:
> And added as a member of https://copr.fedorainfracloud.org/groups/g/theforeman/coprs/?

This part should only be required for building packages which our jobs all handle. For releasing, we just download packages from Copr which anybody can do.

Reply:
But you need to perform actions for branching, right?

Reply:
> But you need to perform actions for branching, right?

Ahh yes, I was focused on just the release doc. Let me add that.